src/main.c

```diff
@@ -71,6 +71,12 @@ int main (int argc, char* argv[])
 	if(strcmp(username, "pihole") != 0)
 		logg("WARNING: Starting pihole-FTL as user %s is not recommended", username);
 
+	// Delay startup (if requested)
+	// Do this before reading the database to make this option not only
+	// useful for interfaces that aren't ready but also for fake-hwclocks
+	// which aren't ready at this point
+	delay_startup();
```

Comment:
Is there any value in putting this before `read_FTLconf` or even `init_shmem`, or even the first line of `main()`?

Reply:
We need it after `read_FTLconf` because this is where we read the value how long to wait ;-)